timeless/reservations/views.py

```diff
@@ -18,13 +18,38 @@
 
 
 class ReservationsListView(views.ListView):
```

Comment:
@vladarefiev shouldn't you add a puzzle to secure these views? At the moment I'm not sure they are secured -- as I remember, they should extend ``SecuredView`` :D

Reply:
@amihaiemil I don't know, I don't implement these views in this PR, just fix them. Let's ask @emilianodellacasa

Reply:
@vladarefiev @amihaiemil there is already a puzzle to secure views, let's not create duplicates